mlrun/config.py

```diff
@@ -117,6 +117,7 @@
         "url": "",
         "type": "standalone",  # deprecated.
     },
+    "sql_db_path_string": "",
```

Comment:
let's keep the same structure as redis - having a hirerchy of it's own and using "url"